icons/folder_type_private.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><title>folder_type_private</title><path d="M27.5,5.5H18.2L16.1,9.7H4.4V26.5H29.6V5.5Zm0,4.2H19.3l1.1-2.1h7.1Z" style="fill:#c15368"/><path d="M20.4,10A10.5,10.5,0,1,0,30.9,20.5,10.5,10.5,0,0,0,20.4,10Zm0,18.375a7.862,7.862,0,0,1-6.41-12.432L24.958,26.911A7.826,7.826,0,0,1,20.4,28.375Zm6.411-3.318L15.843,14.09A7.862,7.862,0,0,1,26.811,25.057Z" style="fill:#d80027"/></svg>
```

Comment:
I chose this icon wth 2 criteria.
1. The simplicity of the icon. IT was the most flat I found. At first, I had also the `public` icon in the background but because it was not distinguishable I removed it.
2. The `brain wiring` as you so call it. I believe this icon serves its purpose.

My suggestions are:
1. Let's release it and await user feedback.
or
2. Submit your version.